test/todo-app/src/components/todo-item/todo-item.tsx

```diff
@@ -7,16 +7,16 @@ export class TodoItem {
 
   @Prop() checked: boolean;
   @Prop() text: string;
-  @Prop() index: number;
   @Event() itemCheck: EventEmitter;
   @Event() itemRemove: EventEmitter;
 
   render() {
+    const {checked, text, itemCheck, itemRemove} = this;
```

Comment:
Whoa, this is an interesting approach to reduce repetition of 'this' and size. 👍